tempesta_fw/http.h

```diff
@@ -353,6 +353,12 @@ bool tfw_http_parse_terminate(TfwHttpMsg *hm);
 int tfw_http_msg_process(void *conn, struct sk_buff *skb, unsigned int off);
 unsigned long tfw_http_req_key_calc(TfwHttpReq *req);
 
+/* Export HTTP connection hooks (for TLS) */
+int tfw_http_conn_init(TfwConnection *);
+void tfw_http_conn_drop(TfwConnection *);
+void tfw_http_conn_release(TfwConnection *);
+TfwMsg *tfw_http_conn_msg_alloc(TfwConnection *);
```

Comment:
The HTTP functions must not be exported this way, rather GFSM state switches must be used instead. See other comments for proposed solution.